scheds/rust/scx_mitosis/src/bpf/mitosis.bpf.c

```diff
@@ -507,7 +495,8 @@ s32 BPF_STRUCT_OPS(mitosis_select_cpu, struct task_struct *p, s32 prev_cpu,
 		goto out;
 	}
 
-	if (tctx->cpumask && bpf_cpumask_empty((const struct cpumask *)tctx->cpumask)) {
+	if (tctx->cpumask &&
+	    bpf_cpumask_empty((const struct cpumask *)tctx->cpumask)) {
```

Comment:
I think you can use [`cast_mask`](https://github.com/sched-ext/scx/blob/05420aa9430388206f10442753a0080647aa969f/scheds/include/scx/common.bpf.h#L446) here